README.md

```diff
@@ -1,73 +1,178 @@
-      "type": "camera",
-      "model": "viam:luxonis:oak-d"
-    }
-  ]
+  "cam_name": "my-oak",
+  "input_source": "color",
+  "num_threads": 2,
+  "num_nce_per_thread": 1,
+  "yolo_config": {
+    "blob_path": "<local-path-to-yolo-model>",
+    "label_map": ["person", "apple", "orange", "tv", "etc."],
+    "confidence_threshold": 0.5,
+    "iou_threshold": 0.5,
+    "anchors": [0, 0, 1, 1, 2, 2, 3, 3, 4, 4],
+    "anchor_masks": {
+      "side26": [1, 2, 3],
+      "side13": [3, 4, 5]
+    },
+    "coordinate_size": 4
+  }
 }
 ```
 
+#### Attributes
+
+The following attributes are available for the `yolo-detection-network` service:
+
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `cam_name` | string | Required | The name of the Viam OAK camera component to set up the service on. |
+| `input_source` | string | Required | The socket name i.e. "cam_a", "cam_b", "cam_c" to get neural network input frames from. Can also be set as "color" to select the primary color sensor on the underlying OAK component. |
+| `num_threads` | int | Optional | How many threads should the DepthAI node use to run the network. Can be 0, 1, or 2, where 0 is auto. Default `1` |
+| `num_nce_per_thread` | int | Optional | How many Neural Compute Engines should a single thread use for inference. Default `1` |
+| `yolo_config` | struct | **Required** | A struct mapping of strings to values representing the sub-configuration for the YOLO model. |
+
+The below attributes are nested inside the `yolo_config` struct:
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `blob_path` | string | Required | The local path to the YOLO model blob. The model must be in the .blob format compatible with Luxonis VPUs. See [here](https://docs.luxonis.com/software/ai-inference/conversion/) for more information. |
+| `labels` | list[string] | Required | A list of strings representing each label the YOLO model can output. Order matters here as YOLO models use label index mapping to retrieve string labels. |
+| `confidence_threshold` | float | Required | The minimum confidence level required for a detection to be considered valid. Default `0.5`. |
+| `iou_threshold` | float | Required | The Intersection Over Union (IOU) threshold used for non-max suppression to filter out overlapping bounding boxes. Default `0.5`. |
+| `anchors` | list[float] | Optional | A list of floats representing the anchor boxes used by the YOLO model. The values should be in pairs representing width and height. Default: `[]` |
+| `anchor_masks` | dict | Optional | A dictionary where keys are strings (e.g., "side26", "side13") representing different scales, and values are lists of integers representing the anchor indices used at each scale. Default: `{}` |
+| `coordinate_size` | int | Optional | The number of coordinates used for each bounding box. Typically, this is 4 for (x, y, width, height). Default `4`. |
+
+## Next Steps
```

Comment:
```suggestion
## Next steps
```

Reply:
Might want to add some actual next steps here besides debugging, like describing how to test the camera in the viam app? quick description would be helpful

Reply:
Added how to view outputs in App